database/migrations/20190625083341_createArtistTable.js

```diff
@@ -0,0 +1,28 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('artist', (artist) => {
```

Comment:
Check the artist object we talked about this morning = https://lambdaschoolstudents.slack.com/archives/GKSFUFP5W/p1561447427001200

artist: {
        id: '',
        artistName: '',
        userDescription: '',
        profilePictureUrl:'',
        username: '',
        email: '',
        password: '',
    }

Reply:
resolved